craft_application/services/provider.py

```diff
@@ -187,3 +190,19 @@ def _get_lxd_provider(self) -> LXDProvider:
     def _get_multipass_provider(self) -> MultipassProvider:
         """Get the Multipass provider for this manager."""
         return MultipassProvider()
+
+    def _capture_logs_from_instance(self, instance: craft_providers.Executor) -> None:
+        """Fetch the logfile from inside `instance` and emit its contents."""
+        source_log_path = pathlib.PosixPath(f"/root/{self._app.name}.log")
```

Comment:
yes we can. I'm leaning towards an utility function that receives the app metadata as parameter, does that sound ok?

Reply:
whew thanks a lot for pointing that out, turns out I had put in the wrong path (/root instead of /tmp)
... and now I don't know how my manual test worked, if the provider and the application were referring to different locations

Reply:
build rockcraft again and things still seem to be working :woman_shrugging: